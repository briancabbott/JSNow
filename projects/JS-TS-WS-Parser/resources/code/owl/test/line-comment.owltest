x x x .hello there!
x x .asdf
.
---
.line-comment-token '.'
input = 'x'*
