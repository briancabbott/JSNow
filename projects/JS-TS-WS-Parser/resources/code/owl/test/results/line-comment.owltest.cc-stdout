input (0 - 23)
