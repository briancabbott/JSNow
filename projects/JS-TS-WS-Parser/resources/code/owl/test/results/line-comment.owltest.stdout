. x x x 
  input-

. x x
  ---
